### PR TITLE
Codeowners, pt.2 

### PR DESCRIPTION
**Pull request by pepyakin**
Continuation of https://github.com/paritytech/substrate/pull/3371

Some people agreed to participate in this experiment, namely:

- @andresilva
- @tomusdrw
- @Robbepop
- @thiolliere 
- @jimpo
- @kianenigma 
- @DemiMarie-parity 

Feel free to ask here or edit the file if you are down to be notified if some parts of the codebase are changed.

### CODE REVIEW

**Comment by thiolliere** [comment]
Support section added

File: CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1,11 @@
-# Lists some code owners
+# Lists some code owners.
+#
+# A codeowner just oversees some part of the codebase. If an owned file is changed then the
+# corresponding codeowner receives a review request. An approval of the codeowner is
+# not required for merging a PR though.
```
Comment (tomusdrw) [inline CODEOWNERS]: :+1:

File: CODEOWNERS [hunk 2]
```diff
@@ -11,5 +18,41 @@
...
+/core/offchain/ @tomusdrw
+/sr-io/src/offchain/ @tomusdrw
+
+# Everything that has RPC in it
+/core/rpc/ @tomusdrw
+/node/rpc/ @tomusdrw
+/node/rpc-client/ @tomusdrw
+/core/rpc-servers/ @tomusdrw
+
+# Grandpa, BABE, consensus stuff
```
Comment (andresilva) [inline CODEOWNERS]:
```suggestion
# GRANDPA, BABE, consensus stuff
```

Comment (pepyakin) [reply CODEOWNERS]: Oh, pardon!

**Pull request merged**